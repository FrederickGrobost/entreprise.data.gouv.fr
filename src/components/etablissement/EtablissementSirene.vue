<template>
  <div class="company">
    <etablissement-links />

    <h3>Informations de la base SIRENE de l'INSEE</h3>
    <div class="company-container">
      <etablissement-sirene-contact />
      <etablissement-sirene-info />
    </div>
  </div>
</template>

<script>
import EtablissementLinks from "@/components/etablissement/etablissementSirene/EtablissementLinks";
import EtablissementSireneContact from "@/components/etablissement/etablissementSirene/EtablissementSireneContact";
import EtablissementSireneInfo from "@/components/etablissement/etablissementSirene/EtablissementSireneInfo";

export default {
  name: "EtablissementSirene",
  components: {
    EtablissementLinks: EtablissementLinks,
    EtablissementSireneContact: EtablissementSireneContact,
    EtablissementSireneInfo: EtablissementSireneInfo
  }
};
</script>

<style lang="scss" scoped>
h3 {
  margin-bottom: 0;
}
.company {
  margin-top: 2em;
}

.panel + .panel {
  margin-left: 2em;
}

@media (max-width: $tablet) {
  .panel + .panel {
    margin-left: 0;
  }
}
</style>
