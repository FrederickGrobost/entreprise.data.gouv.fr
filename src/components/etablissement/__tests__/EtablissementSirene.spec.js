import { createLocalVue, shallowMount } from "@vue/test-utils";
import { __createMocks as createStoreMocks } from "@/store/index.js";
import Vuex from "vuex";
import EtablissementSirene from "@/components/etablissement/EtablissementSirene.vue";

const localVue = createLocalVue();
localVue.use(Vuex);
jest.mock("@/store/index.js");
jest.mock("mapbox-gl", () => ({
  supported: jest.fn(),
  Popup: jest.fn()
}));
describe("EtablissementSirene.vue", () => {
  // let storeMocks
  // let wrapperEtablissement
  // let etablissement
  // beforeEach(() => {
  //   storeMocks = createStoreMocks()
  //   wrapperEtablissement = shallowMount(EtablissementSirene, {
  //     localVue,
  //     store: storeMocks.store
  //   })
  //   etablissement = wrapperEtablissement.vm
  // })
  // TODO
  // test('Computed value coordinates returns the coordinates if both longitude and latitude exists')
});

describe("EtablissementSirene.vue Snapshot testing", () => {
  const storeMocks = createStoreMocks();

  const wrapperES = shallowMount(EtablissementSirene, {
    localVue,
    store: storeMocks.store
  });

  test("It match the snapshot", () => {
    expect(wrapperES.vm.$el).toMatchSnapshot();
  });
});
