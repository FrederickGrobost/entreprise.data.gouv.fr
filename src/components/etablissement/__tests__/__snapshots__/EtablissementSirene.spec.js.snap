// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EtablissementSirene.vue Snapshot testing It match the snapshot 1`] = `
<div
  class="company"
>
  <etablissement-links-stub />
   
  <h3>
    Informations de la base SIRENE de l'INSEE
  </h3>
   
  <div
    class="company-container"
  >
    <etablissement-sirene-contact-stub />
     
    <etablissement-sirene-info-stub />
  </div>
</div>
`;
